cryoet_data_portal_neuroglancer/precompute/instance_mesh.py

```diff
@@ -0,0 +1,122 @@
+import logging
+import typing
+from typing import Any
+
+import numpy as np
+import trimesh
+from tqdm import tqdm
+
+from cryoet_data_portal_neuroglancer.precompute.mesh import decimate_mesh
+from cryoet_data_portal_neuroglancer.utils import rotate_and_translate_mesh
+
+LOGGER = logging.getLogger(__name__)
+
+
+def encode_oriented_mesh(
+    input_geometry: trimesh.Scene | trimesh.Trimesh,
+    data: list[dict[str, Any]],
+    max_lod: int = 2,
+    max_faces_for_first_lod: int = 5_000_000,
+    decimation_aggressiveness: float = 4.5,
```

Comment:
nit: Is this supposed to be `5.0`?

Reply:
Thanks Manasa, forgot to update the docstring! Will fix the docstring and leave the default here at 4.5